pages/entry.jsx

```diff
@@ -5,7 +5,8 @@ import ProjectCardDetialed from '../components/project-card/project-card-detaile
 import Service from '../js/service.js';
 import Utility from '../js/utility.js';
 
-const NO_ENTRY_TITLE = `Entry unavailable`;
+const NO_ENTRY_OG_TITLE = `Entry unavailable`;
+const NO_ENTRY_OG_DESCRIPTION = `Description for this entry is currently unavailable`;
```

Comment:
We should grab this value from the actual entry object in follow-up. I've filed #749 for this